src/melee.cpp

```diff
@@ -1624,69 +1555,8 @@ std::string player::melee_special_effects(Creature &t, damage_instance &d, const
-    const bool stab = d.type_damage(DT_STAB) > d.type_damage(DT_CUT);
-    int used_skill = stab ? get_skill_level( skill_stabbing ) : get_skill_level( skill_cutting );
-    if( has_active_bionic("bio_cqb") ) {
-        used_skill = BIO_CQB_LEVEL;
-    }
-
-    int cutting_penalty = roll_stuck_penalty( stab, tec );
-    // Some weapons splatter a lot of blood around.
-    // TODO: this function shows total damage done by this attack, not final damage inflicted.
-    if (d.total_damage() > 10 && weapon.has_flag("MESSY") ) { //Check if you do non minor damage
-        cutting_penalty /= 6; // Harder to get stuck
-        g->m.add_splash( t.bloodType(), tarpos, 1, 3 );
-    }
-    // Getting your weapon stuck
-    ///\EFFECT_STR decreases chance of getting weapon stuck
-    if (!unarmed_attack() && cutting_penalty > dice(str_cur * 2, 20) && !t.is_hallucination()) {
-        dump << string_format(_("Your %1$s gets stuck in %2$s, pulling it out of your hands!"),
-                              weapon.tname().c_str(), target.c_str());
-        // TODO: better speed debuffs for target, possibly through effects
-        if (monster *m = dynamic_cast<monster *>(&t)) {
-            m->add_item(remove_weapon());
-        } else {
-            // Happens if 't' is not of 'monster' origin (attacking an NPC)
-            g->m.add_item_or_charges( tarpos, remove_weapon(), 1 );
-        }
-
-        t.mod_moves(-30);
-        if (weapon.has_flag("HURT_WHEN_PULLED") && one_in(3)) {
-            //Sharp objects that injure wielder when pulled from hands (so cutting damage only)
-            dump << std::endl << string_format(_("You are hurt by the %s being pulled from your hands!"),
-                                               weapon.tname().c_str());
-            auto hand_hurt = one_in(2) ? bp_hand_l : bp_hand_r;
-            deal_damage( this, hand_hurt, damage_instance::physical( 0, weapon.damage_melee( DT_CUT ) / 2, 0) );
-        }
-    } else {
-        // Approximately "this would kill the target". Unless the target has limb-based hp.
-        if( d.total_damage() > t.get_hp() ) {
-            cutting_penalty /= 2;
-            ///\EFFECT_STABBING reduces chance of weapon getting stuck, if STABBING>CUTTING
-
-            ///\EFFECT_CUTTING reduces chance of weapon getting stuck, if CUTTING>STABBING
-            cutting_penalty -= rng( used_skill, used_skill * 2 + 2 );
-        }
-        if( cutting_penalty >= 50 && !t.is_hallucination() ) {
-            dump << string_format(_("Your %1$s gets stuck in %2$s but you yank it free!"), weapon.tname().c_str(),
-                                  target.c_str());
-        }
-
-        if( weapon.has_flag("SPEAR") ) {
-            ///\EFFECT_STABBING increases damage when pulling out a stuck spear
-            const int stabbing_skill = get_skill_level( skill_stabbing );
-            d.add_damage( DT_CUT, rng( 1, stabbing_skill ) ); //add some extra damage for pulling out a spear
-            t.mod_moves(-30);
```

Comment:
Move draining part of spears is one of the features I really liked.
Still, it wasn't really balanced, especially after spears got their reach attacks.

Might be worth resurrecting in some other form.